packages/nextjs/components/scaffold-eth/Contract/MethodSelector.tsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
+import { Abi } from "viem";
+import { ChevronDownIcon, ChevronRightIcon, XMarkIcon } from "@heroicons/react/24/outline";
+
+type AbiFunction = Extract<Abi[number], { type: "function" }>;
+
+type AbiFallback = Extract<Abi[number], { type: "fallback" }>;
```

Comment:
Same we could import it from abitype, also do we really need this 😅 I just came to know that `fallback` function can also take in on param https://docs.soliditylang.org/en/latest/contracts.html#fallback-function

But I think for simplicity's sake lets remove it for now and create a new issue so that we can handle it properly later, what do you say since any which way we are not showing `fallback` functions currently?